indexing-service/src/main/java/org/apache/druid/indexing/common/task/NoopTask.java

```diff
@@ -41,6 +41,7 @@
  */
 public class NoopTask extends AbstractTask implements PendingSegmentAllocatingTask
 {
+  public static final String TYPE = "noop";
```

Comment:
We can also change `getType()` to return `TYPE` instead of the hardcoded value

Reply:
good catch and fixed.